Packages/Ethereum/Sources/Ethereum/EthRpc1.swift

```diff
@@ -334,6 +334,9 @@ public enum EthRpc1 {
         /// from address
         public var from: EthRpc1.Data? = nil
 
+        enum CodingKeys: String, CodingKey {
```

Comment:
are these used somewhere?

Reply:
they are automatically picked up by Swift JSON Encoder, so only keys listed here will be encoded ('type' is not encoded)